src/net/sourceforge/kolmafia/request/BasementRequest.java

```diff
@@ -526,7 +520,7 @@ private static boolean checkForElementalTest(boolean autoSwitch, final String re
     }
 
     BasementRequest.changeBasementOutfit("element");
-    BasementRequest.canHandleElementTest(autoSwitch, true);
```

Comment:
Was this supposed to change?

Reply:
Yes, as suggested by lint.  

In context

    if (!autoSwitch) {
      return true;
    }

    BasementRequest.changeBasementOutfit("element");
    BasementRequest.canHandleElementTest(true, true);
    return true;
    
 so the changed line will never be executed unless autoSwitch is true.  I couldn't make a case that the change reduced comprehension so I accepted it.